metrics/lib/common.bash

```diff
@@ -295,12 +296,15 @@ wait_ksm_settle(){
 	for ((t=0; t<$1; t++)); do
 
 		newscan=$(cat /sys/kernel/mm/ksm/full_scans)
+		newpages=$(cat /sys/kernel/mm/ksm/pages_shared)
+		[[ "$newpages" -eq 0 ]] && echo "No need to wait for KSM to settle" && return
+
 		if (( newscan != oldscan )); then
 			echo -e "\nnew full_scan ($oldscan to $newscan)"
 
```

Comment:
@grahamwhaley if I move the newpages check to 0 here at the 'newscan' loop the comparison even if I got a 0 value for newscan the loop will not end and it will timeout waiting for KSM to settle, that is the reason why I put it before

Reply:
Ah, OK - understood.
In that case, let's land as is, and see how it goes...